packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -1050,3 +1053,37 @@ class SelectableDataList extends React.Component {
   }
 }
 ```
+
+### Controlling text
+```js
+import React from 'react';
+import {
+  Button,
+  DataList,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+  DataListCheck,
+  DataListAction,
+  truncate,
+  breakWord
```

Comment:
Remove these unused and non-existant imports. You could import `WrapModifier` and then use `WrapModifier.truncate` if you wanted.